### PR TITLE
Cards now render with individual yearly data

### PR DESCRIPTION
**Pull request by Obleo33**

### CODE REVIEW

File: src/DistrictCards/DistrictCards.js [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
+            <h2 className="district-name">{value.location}</h2>
+            <DataLoop yearlyData={value.yearlyData}/>
           </div>
         )
       })}
     </div>
   )
 }
 
+const DataLoop = (yearlyData) => {
```
Comment (Obleo33) [inline src/DistrictCards/DistrictCards.js:18]: We should break this out into its own file

File: src/DistrictCards/DistrictCards.js [hunk 2]
```diff
@@ -1,21 +1,30 @@
...
+  return (
+    <div className="yearly-data">
+      {Object.keys(yearlyData.yearlyData).map((year, index) => {
```
Comment (Obleo33) [inline src/DistrictCards/DistrictCards.js:21]: Our file paths could use some refactoring

File: src/DistrictCards/DistrictCards.js (hunk 1 above)
Comment (becs919) [reply src/DistrictCards/DistrictCards.js:18]: great idea!

File: src/DistrictCards/DistrictCards.js (hunk 2 above)
Comment (becs919) [reply src/DistrictCards/DistrictCards.js:21]: agreed

**Comment by becs919** [comment]
looks good!

**Pull request merged**